vj4/handler/contest.py

```diff
@@ -402,40 +412,22 @@ class ContestEditHandler(base.Handler, ContestStatusMixin):
     tdoc = await contest.get(self.domain_id, tid)
     if not self.own(tdoc, builtin.PERM_EDIT_CONTEST_SELF):
       self.check_perm(builtin.PERM_EDIT_CONTEST)
-    if self.is_live(tdoc) or self.is_done(tdoc):
-      if begin_at_date:
-        raise error.ValidationError('begin_at_date')
-      if begin_at_time:
-        raise error.ValidationError('begin_at_time')
-      begin_at = tdoc['begin_at']
-    else:
-      try:
-        begin_at = datetime.datetime.strptime(begin_at_date + ' ' + begin_at_time, '%Y-%m-%d %H:%M')
-        begin_at = self.timezone.localize(begin_at).astimezone(pytz.utc).replace(tzinfo=None)
-      except ValueError as e:
-        raise error.ValidationError('begin_at_date', 'begin_at_time')
+    try:
```

Comment:
These code removed checkings for contest edit in previous PRs because we have implemented `recalc_status`.